vagrant/roles/candlepin-user/defaults/main.yml

```diff
@@ -7,4 +7,4 @@ candlepin_user_home:  /home/vagrant
 candlepin_git_repo: https://github.com/candlepin/candlepin.git
 candlepin_checkout: /vagrant
 
-candlepin_deploy_args: "-g -a"
+candlepin_deploy_args: "-g -a -t"
```

Comment:
If someone doesn't want to deploy the test data, how would they do so?

Reply:
Good point. My change is overkill. I will only change `README.md` with instruction how to deploy with test data.

Reply:
That would be great.